util/rlp/fuzz/fuzz_targets/untrusted.rs

```diff
@@ -0,0 +1,11 @@
+#![no_main]
+#![cfg(feature = "nightly")]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate rlp;
+
+use rlp::UntrustedRlp;
+
+fuzz_target!(|data: &[u8]| {
+    let _ = UntrustedRlp::new(data);
```

Comment:
> What I'm suggesting is that we have the fuzzer try and generate data which we treat as a bunch of nested lists of bytes (and try to walk accordingly)

Currently, I'm using the `gen_untrusted_corpus.rs` tool to build some basic, well-structured RLP streams. I'm still working on generating all the data types `UntrustedRlp` can normally handle, and then I'll build some RLP streams with mostly good, but a little messed up data. Hopefully, this will give the fuzzer more to mutate over, and we'll get deeper coverage.

> Isn't this the usual strategy with fuzz testing? e.g. if you have a PNG decoder, you try to have the fuzzer generate data that breaks the decoder, not to make the fuzzer generate a picture of a horse.

Absolutely agree. I'm trying to add more to the fuzzer corpus, so that its mutation algorithms can be more efficient in breaking the RLP decoder. Like sending in a PNG of a horse with a trojan encoded inside :)